pxtsim/runtime.ts

```diff
@@ -285,11 +285,13 @@ namespace pxsim {
             return isPxtElectron() || isIpcRenderer();
         }
 
+        export function testLocalhost(url: string): boolean {
```

Comment:
Why can't we just use the browserutils function for this here?

Reply:
I tried that! browserutils is not available in pxtsim. You'll find many functions and types are duplicated in pxtsim.